node/tests/integration_tests.rs

```diff
@@ -0,0 +1,195 @@
+    }
+}
+
+fn get_timestamp() -> Timestamp {
+    std::time::SystemTime::now()
+        .duration_since(std::time::UNIX_EPOCH)
+        .unwrap()
+        .as_millis() as Timestamp
+}
+
+async fn sleep(ms: u64) {
+    tokio::time::sleep(std::time::Duration::from_millis(ms)).await;
+}
+
+async fn run_command(command: &str) {
+    let mut child = tokio::process::Command::new("sh")
+        .arg("-c")
+        .arg(command)
+        .spawn()
+        .expect("failed to execute process");
+    let ecode = child.wait().await.expect("failed to wait on child");
+    assert!(ecode.success());
+}
+
+async fn setup_genesis_repo(path: &str, rs: ReservedState) {
+    run_command(&format!("cd {}", path)).await;
+    run_command("mkdir repo").await;
+    run_command("cd repo && git init").await;
+    run_command("git commit --allow-empty -m 'initial'").await;
+    simperby_repository::raw::reserved_state::write_reserved_state("./", &rs)
+        .await
+        .unwrap();
+    run_command("git add -A").await;
+    run_command("git commit -m 'genesis'").await;
+}
+
+// TODO: remove this
+async fn push_repo(path: &str) {
+    run_command(&format!("cd {}", path)).await;
+    run_command("cd repo").await;
+    run_command("git push public_1 -f --all").await;
+}
+
+#[ignore]
+#[tokio::test]
+async fn normal_1() {
+    let configs = (0..4)
+        .map(|i| generate_config("normal_1".to_owned(), i))
+        .collect::<Vec<_>>();
+    let genesis_header = BlockHeader {
+        author: configs[0].public_key.clone(),
+        prev_block_finalization_proof: vec![],
+        previous_hash: Hash256::zero(),
+        height: 0,
+        timestamp: get_timestamp(),
+        commit_merkle_root: Hash256::zero(),
+        repository_merkle_root: Hash256::zero(),
+        validator_set: configs.iter().map(|c| (c.public_key.clone(), 1)).collect(),
+        version: "0.1.0".to_owned(),
+    };
+    let rs = ReservedState {
+        genesis_info: GenesisInfo {
+            header: genesis_header.clone(),
+            genesis_proof: configs
+                .iter()
+                .map(|c| {
+                    TypedSignature::new(
+                        Signature::sign(genesis_header.to_hash256(), &c.private_key).unwrap(),
+                        c.public_key.clone(),
+                    )
+                })
+                .collect(),
+            chain_name: "normal_1".to_owned(),
+        },
+        members: configs
+            .iter()
+            .enumerate()
+            .map(|(i, c)| Member {
+                name: format!("node-{}", i),
+                public_key: c.public_key.clone(),
+                governance_voting_power: 1,
+                consensus_voting_power: 1,
+                governance_delegations: None,
+                consensus_delegations: None,
+            })
+            .chain(std::iter::once(Member {
+                name: "node-relayer".to_owned(),
+                public_key: configs[0].public_key.clone(),
```

Comment:
is it possible for a member to perform a role as a node and also a relayer node at the same time?